ui/src/apps/searchLogs/components/SearchHistory.tsx

```diff
@@ -134,20 +127,17 @@ export default function SearchHistory() {
   }
 
   const historyList: History[] = taskGroups.map((taskGroup) => {
```

Comment:
it is totally fine not to map the taskgroup now, since there is no need to compute something complex.

Reply:
I have a little confusion about this.

Here is a TypeScript type `History` as model for rendering `CardTable`. 

Line 129 to 143 will mapping the data from `LogserachTaskGroupModel` to `History`, need using `parseHistoryStatsParams()` to transfer some fields and resolve default values.

You means  using `LogserachTaskGroupModel` directly instead of `History`?

Reply:
Yes, I think it is totally fine to remove `History` and just use `LogserachTaskGroupModel`.